_includes/selector.html

```diff
@@ -9,6 +9,7 @@
     .selector-bg {
         background-color: #9943ff;
         padding: 1rem;
+        border-radius: 5px;
```

Comment:
I'm sometimes a fan of rounded rectangles but I think this selector looked fine without it. No strong feelings; it's fine either way and I will avoid bikeshedding this further. :)

Reply:
I don't think it looked _bad_ or anything, but was inconsistent with the rest of the page since all other buttons and code blocks have rounded corners and I like consistency and it was a quick change.